dev-games/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="Simulator Construction Toolkit"
+DESCRIPTION="Set of open-source libraries designed as building blocks \
+for quickly assembling 3D simulations, games, and visualization applications."
+HOMEPAGE="https://home.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
+PATCHES="simgear-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion
+	lib:libSimGearCored$secondaryArchSuffix = $portVersion
+	lib:libSimGearScened$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
```

Comment:
```diff
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
 	lib:libgl$secondaryArchSuffix
```
Found with `hp -S simgear`:
```
Warning: POLICY ERROR: "/packaging/simgear/lib/libSimGearCored.so.2018.2.2" needs library "libcurl.so.4", but the package doesn't seem to declare that as a requirement
Warning: POLICY ERROR: "/packaging/simgear/lib/libSimGearScened.so.2018.2.2" needs library "libcurl.so.4", but the package doesn't seem to declare that as a requirement
Warning: Error: packaging policy violation(s) in strict mode
Error: Build has failed - stopping.
```

BTW, it seems my attempt to have the header files installed in the correct location only works for a few files. Most headers are still being installed in the default directory...